src/v2/providers/pubsub.ts

```diff
@@ -6,7 +6,8 @@ import * as options from '../options';
 /**
  * Interface representing a Google Cloud Pub/Sub message.
  *
- * @param data Payload of a Pub/Sub message.
+ * @param data - Payload of a Pub/Sub message.
```

Comment:
document the type param

Reply:
Updated... but there is likely a much better definition.